scripts/test/Muon/grouping_tab/pairing_table_group_selector_test.py

```diff
@@ -26,9 +30,14 @@ def setUp(self):
         # Store an empty widget to parent all the views, and ensure they are deleted correctly
         self.obj = QtGui.QWidget()
 
-        self.data = MuonDataContext()
+        self.loaded_data = MuonLoadData()
+        self.data_context = MuonDataContext(self.loaded_data)
+        self.gui_context = MuonGuiContext()
+        self.group_context = MuonGroupPairContext()
+        self.context = MuonContext(muon_data_context=self.data_context, muon_group_context=self.group_context,
```

Comment:
since we are going to be doing these few lines a lot is it worth making a file with the function 

```def set_up_context(self):
        self.loaded_data = MuonLoadData()
        self.data_context = MuonDataContext(self.loaded_data)
        self.gui_context = MuonGuiContext()
        self.group_context = MuonGroupPairContext() ```

Reply:
I think you are correct here I'll move these lines out to a common place.